### PR TITLE
Update TileDB-Cloud-R with respect to current openapi-v1.yaml

### PR DESCRIPTION
**Pull request by johnkerl**
Current API spec: https://github.com/TileDB-Inc/TileDB-Cloud-API-Spec/blob/master/openapi-v1.yaml

Components of this PR:

* Mostly, the results of running autogen, as described at https://github.com/TileDB-Inc/TileDB-Cloud-R/blob/master/howToBuild.md. This is the first commit on this PR.
* A few manual edits on top of the autogen, marked as such for the benefit of future autogen runs. These are all marked `MANUAL EDIT`. Also, iterative work on the new manual-layer file `R/execute.R`. This is the second commit on this PR.
* More unit-test cases can and will be added on a subsequent PR (or, on this one if desired).
* There also be follow-on work for multiple arrays, parameterized `layout`, and more.

A current example (pending more docwork to come):

```
library(tiledbcloud)
library(tiledb)

tiledbcloud::login(username=username, password=password,host=host)

myfunc <- function(df) {
  vec <- as.vector(df[["a"]])
  list(min=min(vec), med=median(vec), max=max(vec))
}

result <- tiledbcloud::execute_array_udf(
  namespace="demo",
  array="quickstart_dense",
  udf=myfunc,
  selectedRanges=list(cbind(1,2), cbind(1,2)),
  attrs=c("a")
)
print(result)
```

with output (server-side support PRs not yet merged)

```
     type value
1 integer     1
2  double   3.5
3 integer     6
```

### CODE REVIEW

**Comment by eddelbuettel** [comment]
Yikes:   image

**Review by antalakas — APPROVED** [review]

**Review by eddelbuettel — APPROVED** [review]
Looks "good" for various definitions of good -- hard to assess a hundred generated R files, a hundred generated .md and a hundred .Rd help files in bulk.

The pkgdown docs in docs/ have vanished. That is fine--we can add them after the fact; you can also add them. I call a two-liner for the R package which is `library(pkgdown); build_site()` (from the package root directory) and which I never committed as it is so tiny.

Lastly, it is good that the package has unit tests.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
-Author: TileDB, Inc.
-Copyright: TileDB, Inc.
-Maintainer: Dirk Eddelbuettel <dirk@tiledb.com>
-Description: The TileDB Cloud Platform API Client Package offers
- access to the TileDB Cloud service.
-License: MIT + file LICENSE
-URL: https://github.com/TileDB-Inc/TileDB-Cloud-R
+Title: R Package Client for TileDB Storage Platform API
+Version: 0.0.3
+Authors@R: person("OpenAPI Generator community", email = "team@openapitools.org", role = c("aut", "cre"))
```
[the remark this answers is not in the document]
Comment (johnkerl) [reply DESCRIPTION:4]: Thanks @eddelbuettel !! There are indeed lots of autogen files.

One improvement point is that as we bring R array UDFs up to parity with Python array UDFs we can run autogen more often, and these PRs will be smaller -- there won't be as much accumulated API-spec change as there was this time.

Secondly, given the huge number of files involved, this is where I especially appreciate your R-expert eyes as you've quite hit the nail on the head with regard to flagging opportunities for manual edits I'd missed.

Super-happy for your review here :D :D :D

[the remark this answers is not in the document]
Comment (johnkerl) [reply DESCRIPTION:4]: Likewise thanks for the pkgdown tip -- I'll add this to the re-autogen checklist file we have in `howToBuild.md`. And likewise the package-ownership notes. :D

[the remark this answers is not in the document]
Comment (johnkerl) [reply DESCRIPTION:4]: Re unit tests -- I will add more for query-ranges, at least, as those are the most sensitive and crucial.

**Pull request merged**